src/hooks/xcm/useXcmBridgeV2.ts

```diff
@@ -262,30 +261,16 @@ export function useXcmBridgeV2(selectedToken: Ref<Asset>) {
   };
 
   const connectOriginChain = async (endpoint: string): Promise<void> => {
-    const shouldConnectApi = (chains: string[]): boolean => {
-      if (isAstarNativeTransfer.value) {
-        return chains.includes(srcChain.value.name) || chains.includes(destChain.value.name);
-      } else {
-        return chains.includes(originChain.value.name);
-      }
-    };
-
-    const shouldConnectMoonbeam = shouldConnectApi([Moonriver.name]);
-    const shouldConnectAcala = shouldConnectApi([Acala.name, Karura.name]);
-
+    if (!endpoint) return;
```

Comment:
any situation if endpoint is missing? also it's duplicate with check function for `apiObj` below

Reply:
Thank you for checking. I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/504/commits/5ffeff1f0e3cb47e395b7202474df8b6037ddd2d#diff-7150ace3457f5b63c5e0c174925d33dc69786ff71dc3498771ba00a6db530704L264) commit.